reactive/kubernetes_worker.py

```diff
@@ -456,10 +472,10 @@ def watch_for_changes():
       'tls_client.ca.saved', 'tls_client.certs.saved',
       'kube-control.dns.available', 'kube-control.auth.available',
       'cni.available', 'kubernetes-worker.restart-needed',
-      'worker.auth.bootstrapped')
+      'worker.auth.bootstrapped', 'endpoint.container-runtime.available')
 @when_not('kubernetes-worker.cloud.pending',
           'kubernetes-worker.cloud.blocked')
-def start_worker(kube_api, kube_control, auth_control, cni):
+def start_worker(kube_api, kube_control, auth_control, cni, runtime):
```

Comment:
We're trying to transition to endpoint_from_flag where we can for clarity. I would suggest it is time to update this function to use that and get runtime via endpoint_from_flag as well.

Reply:
Makes sense; updated!